src/Files.App/Data/Items/ListedItem.cs

```diff
@@ -102,7 +102,7 @@ public string[] FileTags
 				{
 					Debug.Assert(value != null);
 					var dbInstance = FileTagsHelper.GetDbInstance();
-					dbInstance.SetTags(ItemPath, FileFRN, value);
+					dbInstance.SetTags(ItemPath, FileFRN, value ?? []);
```

Comment:
We already asserted `value` is not null. I think we can keep here as it was.

Reply:
`Debug.Assert` is ignored in the release build, so I'm not sure if `value` is not null here.